### ISSUE TITLE
Add support for custom crash and log collection URL (to use own proxies)

### ISSUE THREAD

**Issue by johnfriedrich** [newrelic/newrelic-capacitor-plugin#20]
[NOTE]: # ( ^^ Provide a general summary of the request in the title above. ^^ )

## Summary

When using Android or iOS native agents you can add a custom crash and log collection url. See here:
https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/android-agent-configuration-feature-flags/#logging
e.g.
```
NewRelic.withApplicationToken("YOUR_APP_TOKEN")
.usingCollectorAddress("harvest-upload-delegate.domain.com")<------
.start(this.getApplication());
```

This functionality is missing in this Plugin.

## Desired Behavior

A method overload which takes the collector and crash collector endpoint in addition to the appkey when init the new relic capacitor plugin

If this gets implemented and you update the documentation, one small thing:
- In the readme it mentions to put the init code into the index.tsx, I think thats react. Would be nice to get an example for Angular aswell

Thanks for this plugin!

**Comment by johnfriedrich** [comment]
Just noticed it's already added here but not yet released:
https://github.com/newrelic/newrelic-capacitor-plugin/pull/16

**Comment by kthai-nr** [comment]
Hello, appreciate the feedback. You should be able to put the init code within `main.ts` of your angular app and I'll update the README for that. We hope to push out a new release sometime soon, so I'll notify you via this thread when we do.

**Comment by kthai-nr** [comment]
Hello @johnfriedrich, we have released v1.1.0 that will have agent configuration options for collector endpoints. You can find the release here: https://www.npmjs.com/package/@newrelic/newrelic-capacitor-plugin/v/1.1.0
